src/Doctrine/Odm/Filter/AbstractFilter.php

```diff
@@ -56,11 +60,16 @@ public function apply(Builder $aggregationBuilder, string $resourceClass, ?Opera
      */
     abstract protected function filterProperty(string $property, $value, Builder $aggregationBuilder, string $resourceClass, ?Operation $operation = null, array &$context = []): void;
 
-    protected function getManagerRegistry(): ManagerRegistry
+    protected function getManagerRegistry(): ?ManagerRegistry
```

Comment:
Possibly, but since you suggested making the `ManagerRegistry` nullable ([see code here](https://github.com/api-platform/core/blob/1892e6aeffbcc9c748dd0b3d929090f38f835274/src/Doctrine/Orm/Filter/AbstractFilter.php#L34)), I’ve simply updated the getter to maintain consistency. Otherwise, what happens if the `managerRegistry` property is null and we call `getManagerRegistry`?"

Reply:
Done